server/routes/web.py

```diff
@@ -0,0 +1,28 @@
+from flask import Blueprint, redirect, current_app, Response
+from flask_login import current_user
+from http import HTTPStatus as httpStatus
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+HOST = os.getenv("FLASK_HOST")
+
+bp = Blueprint("webapp", __name__, url_prefix="/app")
+
+
+def frontend_page() -> Response:
+    current_app.send_static_file("index.html"), httpStatus.OK
```

Comment:
It looks like it's missing a `return` statement

Reply:
Good catch! Don't know how it worked for me considering that `return` was missing.